crypto/digestible/derive/Cargo.toml

```diff
@@ -8,9 +8,9 @@ edition = "2018"
 proc_macro = true
 
 [dependencies]
-proc-macro2 = "0.4.4"
-quote = "0.6.3"
-syn = { version = "0.15", features = [ "extra-traits" ] }
+proc-macro2 = "1.0.8"
```

Comment:
There are actually newer versions, but courtesy of `mbedtls` and its dependency on an old `bindgen`, we need to pin to these exact versions :(

Reply:
one thing is, there was a new version of cargo feature that can allow separating dependencies for build-time tools -- I don't know that we ever enabled it but it might help here? i'm not sure

Reply:
https://github.com/rust-lang/cargo/pull/8003

Reply:
it sounds like it might not actually help yet anyways

Reply:
Yeah, this is caused because we can only have one version of `clang-sys` at a time. Right now grpc and mbedtls both need version 6, so that's the one we're stuck on.